openfe/protocols/openmm_rfe/equil_rfe_settings.py

```diff
@@ -169,19 +169,20 @@ class Config:
     or `independent` (independently sampled lambda windows).
     Default `repex`.
     """
-    online_analysis_interval: Optional[int] = None
+    online_analysis_interval: Optional[int] = 200
     """
     Interval at which to perform an analysis of the free energies.
     At each interval the free energy is estimate and the simulation is
     considered complete if the free energy estimate is below
     ``online_analysis_target_error``. If set, will write a yaml file with
     real time analysis data. Default `None`.
     """
-    online_analysis_target_error = 0.1 * unit.boltzmann_constant * unit.kelvin
+    online_analysis_target_error = 0.0 * unit.boltzmann_constant * unit.kelvin
     """
     Target error for the online analysis measured in kT. Once the free energy
     is at or below this value, the simulation will be considered complete.
-    Default 0.1 * unit.bolzmann_constant * unit.kelvin
+    Default 0.0 * unit.bolzmann_constant * unit.kelvin, i.e. no early
+    termination will occur.
```

Comment:
If we can, suggesting a sensible default for when people do want to use this feature would be helpful.  Was 0.1 a reasonable value when using this feature?